spec/factories/review_factory.rb

```diff
@@ -0,0 +1,10 @@
+require 'factory_girl_rails'
+
+FactoryGirl.define do
+  factory :review do
+    rating 5
+    sequence(:comment) { |n| "Best lifehack ever#{n}" }
+    association :lifehack
+    association :creator, factory: :user
+  end
```

Comment:
What's great about factory girl is as long as you have a factory for all of your objects you can actually just change lines 7 and 8 to be: 
```
lifehack
creator
```